library/public/engine.h

```diff
@@ -95,6 +95,10 @@ class F3D_EXPORT engine
    * Supports full path, relative path, and plugin name.
    * On Linux and macOS, uses LD_LIBRARY_PATH to find the plugin.
    * On Windows, the plugin should be located in the same folder as the executable.
+   * The plugin "native" is always available and includes native VTK readers.
+   * If built and available in your build, f3d is providing 4 additional plugins:
+   * "exodus", "occt", "assimp", "alembic"
+   * Custom plugins can also be available that f3d is not supporting officially.
```

Comment:
should we provide a example of a dummy plugin ?

Reply:
Why? The `exodus` plugin is pretty much a dummy plugin.

Reply:
I mean a plugin built outside of F3D

Reply:
ok